specs/merge/beacon-chain.md

```diff
@@ -168,7 +180,9 @@ def process_block(state: BeaconState, block: BeaconBlock) -> None:
     process_randao(state, block.body)
     process_eth1_data(state, block.body)
     process_operations(state, block.body)
-    process_execution_payload(state, block.body)  # [New in Merge]
+    # Pre-merge, skip execution payload processing
+    if is_execution_enabled(state, block):
```

Comment:
What do you think about the following name? I am not sure how idiomatic it is but it reads better to me than "enabled". As the latter sounds more like an externally defined option that is immutable in the runtime.
```suggestion
    if should_process_execution_payload(state, block):
```

Reply:
I *do* like the idea of being able to override the function in a non-merge context to just have execution enabled cleanly from the start. That said, I see your point.

all predicates in the spec currently have an `is_` prefix and so by default, I'd like to keep that unless we can't come up with something better.

that said, I can't currently come up with anything better so I'm happy to use `should_` for now and then see what we come up with in subsequent review